src/types.ts

```diff
@@ -61,7 +61,7 @@ export type InputTextNode = {
 
 type PlusType = [number, string];
 
-type RoamNode = { "db/id": number };
+export type RoamNode = { "db/id": number };
```

Comment:
where is this being used such that we're exporting it? It'll have to be reexported from index if you want to use the type outside of the module

Reply:
ah, yes that was the hope 😅, let me do that